scripts/set-environment

```diff
@@ -66,6 +66,11 @@ fi
   echo 'export BEHAT_ADMIN_PASSWORD=$(openssl rand -base64 24)'
   echo 'export BEHAT_ADMIN_USERNAME=pantheon-ci-testing-$CI_BUILD_NUMBER'
   echo 'export BEHAT_ADMIN_EMAIL=no-reply+ci-$CI_BUILD_NUMBER@getpantheon.com'
+  echo "export MULTIDEV_SITE_URL='https://$TERMINUS_ENV-$TERMINUS_SITE.pantheonsite.io/'"
```

Comment:
$TERMINUS_SITE and $TERMINUS_ENV will not necessarily be defined here (if they are taking default values). This line should therefore be single-quoted.